databricks/koalas/tests/test_repr.py

```diff
@@ -19,7 +19,7 @@
 
 
 class ReprTests(ReusedSQLTestCase):
-    max_display_count = 123
+    max_display_count = 23
```

Comment:
pandas' `display.max_rows` value is 60 by default. So, lowering then that will test the changes correctly.